src/webpack/webpack.ts

```diff
@@ -116,29 +116,42 @@ function _shouldIgnoreValue(value: any) {
     if (value == null) return true;
     if (value === window) return true;
     if (value === document || value === document.documentElement) return true;
-    if (value[Symbol.toStringTag] === "DOMTokenList") return true;
+    if (value[Symbol.toStringTag] === "DOMTokenList" || value[Symbol.toStringTag] === "IntlMessagesProxy") return true;
+    // Discord might export a Proxy that returns non-null values for any property key (e.g. their i18n Proxy (caught by above check))
+    // We can use a unique string to detect this and ignore it
+    if (value["is this a proxy that returns values for any key?"]) return true;
     if (value instanceof TypedArray) return true;
 
     return false;
 }
 
-export function _shouldIgnoreModule(exports: any) {
+function makePropertyNonEnumerable(target: Object, key: PropertyKey) {
+    const descriptor = Object.getOwnPropertyDescriptor(target, key);
+    if (descriptor == null) return;
+
+    Reflect.defineProperty(target, key, {
+        ...descriptor,
+        enumerable: false
+    });
+}
+
+export function _blacklistBadModules(requireCache: NonNullable<AnyWebpackRequire["c"]>, exports: any, moduleId: PropertyKey) {
     if (_shouldIgnoreValue(exports)) {
+        makePropertyNonEnumerable(requireCache, moduleId);
         return true;
     }
 
-    if (typeof exports !== "object") {
+    if (typeof exports !== "object" || exports == null) {
         return false;
     }
 
-    let allNonEnumerable = true;
     for (const exportKey in exports) {
-        if (!_shouldIgnoreValue(exports[exportKey])) {
-            allNonEnumerable = false;
+        if (_shouldIgnoreValue(exports[exportKey])) {
+            makePropertyNonEnumerable(exports, exportKey);
         }
     }
 
-    return allNonEnumerable;
+    return false;
```

Comment:
```suggestion
    return false;
```

should still return true if all are non enumerable